kivy/lib/osc/OSC.py

```diff
@@ -34,6 +34,25 @@
 import pprint
 from kivy.compat import string_types
 
+class binaryString(bytearray):
```

Comment:
good catch. I'm pretty sure that's my original workaround method and then I realized I could just use binarystrings directly and check in OSCArgument for bytearray/string.
Fine to remove.